packages/react-components/src/organisms/ManuscriptVersionCard.tsx

```diff
@@ -108,9 +113,52 @@ const hasAdditionalInfo = (version: ManuscriptVersion) =>
   version.requestingApcCoverage ||
   version.otherDetails;
 
-const ManuscriptVersionCard: React.FC<ManuscriptVersionCardProps> = (
-  version,
+export const getLifecycleCode = (
+  version: Pick<ManuscriptVersion, 'type' | 'lifecycle'>,
```

Comment:
minor suggestion, we can simplify since we only need the lifecycle here
```suggestion
  lifecycle: ManuscriptLifecycle,
```